SUMMARY.md

```diff
@@ -8,7 +8,7 @@
   - [The `Default` Trait](./idioms/default.md)
   - [Collections Are Smart Pointers](./idioms/deref.md)
   - [Finalisation in Destructors](./idioms/dtor-finally.md)
-  - [`mem::replace(_)`](./idioms/mem-replace.md)
+  - [`mem::take(_)`](./idioms/mem-take.md)
```

Comment:
I'm not so sure here if this is the right way. This is not really backwards compatible, so for example people bookmarking this idiom for later, will get a 404 for this now. Not sure if it would be better to keep the name for the markdown file would be better (so links stay the same), but changing the title to `mem::{replace(_), take(_)}` would be more sufficient. Let's discuss this here.

Reply:
I agree. Rebased so the filename change never happened, and changed title to `mem::{take(_), replace(_)}`.